### PR TITLE
🇺🇸 Add California pin

### PR DESCRIPTION
**Pull request by danielsp45**
🇺🇸

### CODE REVIEW

File: data/places.json [hunk 1]
```diff
@@ -106,5 +106,14 @@
...
     "photo": "/photos/jpcorreia99/dublin.jpg"
+  },
+  {
+    "author": "Daniel Pereira",
+    "type": "sticker",
+    "city": "Santa Cruz",
+    "country": "United States of America",
+    "coordinates": [36.955797, -121.972159],
+    "date": "2022-07-04",
+    "photo": "/photos/danielpereira/daniel-california.jpeg"
```
Comment (nelsonmestevao) [inline data/places.json:117]: Can you replace the name to just `california.jpeg`?

Comment (danielsp45) [reply data/places.json:117]: Sim 💪🏻

**Review by nelsonmestevao — APPROVED** [review]
So cool! :clap: 

And on top in the famous 4th of July :tada:

**Comment by danielsp45** [comment]
> So cool! 👏
> 
> And on top in the famous 4th of July 🎉

4th of July is crazy there 🤯

**Pull request merged**